CMSgov/mint-app
# [EASI-4446] Get Model Plan and Operational Solution Data by Operational Solution Key

# EASI-4446


## Changes and Description

- GQL
- SQL
- Models
  - Raw model for deserialization of SQL query
  - Added full constructors for Operational Solution and Model Plan for deserialization purposes
- Updated Postman collection
- Unit tests



## How to test this change

For your convenience, a variety of tests has been added as unit tests.
For manual testing, please use the Postman query `Get Model Plans By Operational Solution Key` and structure data however you see fit.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
